lib/MusicBrainz/Server/Controller/Area.pm

```diff
@@ -413,22 +413,11 @@ sub _merge_load_entities
     $c->model('AreaType')->load(@areas);
 };
 
-=head1 LICENSE
-
-This software is provided "as is", without warranty of any kind, express or
-implied, including  but not limited  to the warranties of  merchantability,
-fitness for a particular purpose and noninfringement. In no event shall the
-authors or  copyright  holders be  liable for any claim,  damages or  other
-liability, whether  in an  action of  contract, tort  or otherwise, arising
-from,  out of  or in  connection with  the software or  the  use  or  other
-dealings in the software.
-
-GPL - The GNU General Public License    http://www.gnu.org/licenses/gpl.txt
-Permits anyone the right to use and modify the software without limitations
-as long as proper  credits are given  and the original  and modified source
-code are included. Requires  that the final product, software derivate from
-the original  source or any  software  utilizing a GPL  component, such  as
-this, is also licensed under the GPL license.
+=head1 COPYRIGHT AND LICENSE
+
+This file is part of MusicBrainz, the open internet music database,
+and is licensed under the GPL version 2, or (at your option) any
+later version: http://www.gnu.org/licenses/gpl-2.0.txt
```

Comment:
It’s missing a copyright notices, which from git history could be:

```pod
Copyright (C) 2013-2021 MetaBrainz Foundation
Copyright (C) 2014 Ulrich Klauer
Copyright (C) 2015 Jesse Weinstein
Copyright (C) 2021 jesus2099
```

Reply:
I found a bunch more of those, I added copyright statements after reviewing the history of each file to see who has made significant additions to it :)

Reply:
That was just an example.

For the record, these external contributors are not credited for that file because their contributions were not significant enough, we all agreed with @bitmap. However, they did significant contributions to other files, and @reosarevok credited them accordingly; See https://github.com/metabrainz/musicbrainz-server/pull/2115/commits/aef02a68b374da43e902918e8d6d7aa2bd92ca0d#diff-b37d435b7fbd0241b783ab9b1e537e80b49ef116714269c755af3a58448e333e for example. Furthermore, all added contributors already had credits elsewhere, so no unwanted credits have been added so far.